guides/common/modules/proc_generating-host-monitoring-reports.adoc

```diff
@@ -52,4 +52,9 @@ You can use the `no limit` value to show all entitlements.
 --inputs "Days from Now=60" \
 --name "Subscription - Entitlement Report"
 ----
-====
\ No newline at end of file
+====
+
+For hosts of non-Simple Content Access (SCA) organizations, a new report template called *Host - Installed Products* is available.
+The template displays installed product information along with other metrics unrelated to subscriptions that were previously displayed on the Subscription - Entitlement Report page.
+The subscription entitlement report no longer lists any hosts that are in SCA organizations.
+It only outputs information for hosts that are in organizations that do not use SCA.
```

Comment:
Shouldn't this be other way around? If there is a new report template from non-SCA orgs, it would make sense that the old report does not have that info anymore.

Reply:
The subscription entitlement report does not list hosts or information in hosts that are in SCA organizations. It only lists information for hosts in organizations that are non-SCA.

Reply:
Perhaps I don't understand this. What is the connection between **Host - Installed Products** and **Subscription - Entitlement Report**?

Reply:
Host - Installed Products is a new report that reports everything in the Subscription - Entitlement Report besides subs information. It's for people using SCA. For people not using SCA, they can keep using the Subscription - Entitlement Report. And now, for hosts in orgs using SCA, those hosts won't show up in the Subscription - Entitlement Report.

Reply:
I think editing the information into a list would help because a list makes comparing the two reports easier. Like this:

> Host - Installed Products:: Use this template for hosts in Simple Content Access (SCA) organizations.
> It generates a report with installed product information along with other metrics included in Subscription - Entitlement Report.
> It only outputs information for hosts in organizations that use SCA.

> Subscription - Entitlement Report:: Use this template for hosts that are not in SCA organization.
> It generates a report with ... information.
> It only outputs information for hosts in organizations that do not use SCA.

If you decide to not go with a list, I'd definitely recommend removing the references to how things worked before ("new template", "no longer lists", "previously displayed"). Those are great for release notes, but a guide should describe things as they are at a given moment.